app/models/miq_alert_status.rb

```diff
@@ -1,4 +1,19 @@
+require 'ancestry'
+
 class MiqAlertStatus < ApplicationRecord
+  SEVERITY_LEVELS = %w(error warning info).freeze
+
+  has_ancestry
   belongs_to :miq_alert
   belongs_to :resource, :polymorphic => true
+  has_many :miq_alert_status_actions, -> { order "created_at" }, :dependent => :destroy
+  virtual_column :assignee, :type => :string
+
+  def assignee
+    miq_alert_status_actions.where(:action_type => %w(assign unassign)).last.try(:assignee)
```

Comment:
@moolitayer no need for explicit ordering here? Is there a guarantee for the actions to be ordered or is it just a coincidence?

Also, to simplify, isn't `unassign` just an `assign` to `nil`? Can't you just use `assign`?